src/content/tutorials/fleet-management/app-platform/_index.md

```diff
@@ -1,11 +1,10 @@
 ---
 linkTitle: App platform
-title: Guides for using the App platform
+title: App management
```

Comment:
It's strange to see a different title in menu vs. on the right side. This page doesn't describe the "App platform".

Suggestion: only `title: App management`

Reply:
You're right, will fix